docker/mint/server.py

```diff
@@ -45,7 +45,7 @@ def send_transaction():
     address = flask.request.args['address']
 
     # Return immediately if address is invalid
-    if re.match('^[a-f0-9]{32}$', address) is None:
+    if re.match('^[a-f0-9]{64}$', address) is None:
```

Comment:
I would still expect this to be 32 bytes... (if that's what the 32 means)...

Reply:
hex encoded version of auth key has len of 64 from what I've seen